inference-engine/tests/functional/plugin/plaidml/shared_tests_instances/single_layer_tests/shuffle_channels.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "single_layer_tests/shuffle_channels.hpp"
+using LayerTestsDefinitions::ShuffleChannelsLayerTest;
+
+namespace {
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+    InferenceEngine::Precision::I32,
+    InferenceEngine::Precision::FP32,
+};
+
+const std::vector<std::vector<size_t>> inputShapes = {
+    {5, 12, 200, 400},
+    {1, 30, 40, 40}
+};
+
+const std::vector<shuffleChannelsSpecificParams> shuffleChannelsParams = {
+    {1, 2},
+    {1, 6}
```

Comment:
I suspect if you changed the first of these to `{3, 2}` you would discover errors from ignoring the `axis` parameter

Reply:
yes, you are right, the axis does matter here. the axis specifies the index of a channel dimension, I used to think our input order may always be 'NCHW'. but it's not true, it could be any form. and axis will indicate which dimension is channel dimension. so in the test case, we may need to match inputShape and shuffleChannelThirdParams.